### DISCUSSION TITLE
I can't see GitHub pages, it is just blank.

### DISCUSSION THREAD

**Discussion by 0dayScorpio** [community/community#88507] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, guys!
I am writing, because when I load some page on GitHub to download and app from, it doesn't show the page at all. It just shows a blank screen with the author of the script and no text, code or whatever.
![Screenshot_2024-01-17_02_04_53](https://github.com/community/community/assets/92162833/936bfee7-389b-494c-a983-7f833e9ff948)
I will be thankful for the support!

**Comment by Shahnawazk7944** [comment]
You might be experiencing an internet connectivity issue. Try reloading the page two to three times, or press Ctrl+Shift+R to forcefully reload the page.

**Reply by 0dayScorpio** [reply]
Thanks, but I did it multiple times and It doesn't work with neither root or kali. I checked the WI-FI connection and the problem isn't there

**Reply by Shahnawazk7944** [reply]
I believe you are using certain extensions that temporarily disable other extensions that could disrupt page loading.

**Reply by Shahnawazk7944** [reply]
if still persist try all the below debugging

1. Outdated data can cause loading problems. Clear your browser's cache and cookies.
2. If the issue persists, switch to a different browser to rule out browser-specific problems.
3. If you're using a VPN or proxy, temporarily disable it to see if it's causing interference.
4. Check if firewall settings or security software are blocking GitHub.

**Reply by Shahnawazk7944** [reply]
OR

1. Right-click the blank page and select "Inspect" or "Inspect Element." Look for errors in the Console tab.
2. Find other GitHub pages to see if the issue is widespread or specific to that page.
3. If the problem persists, report it to GitHub's support team.

**Reply by 0dayScorpio** [reply]
Tried anything and it didn't work. I will contact the support, thanks for your time!

**Reply by Shahnawazk7944** [reply]
check these discussions, is that your problem?

https://github.com/orgs/community/discussions/60881#discussion-5403453
https://github.com/orgs/community/discussions/88596#discussion-6087553

**Comment by WojciechMigda** [comment]
I think you are having the same problem as me, described in this ticket: https://github.com/orgs/community/discussions/86701
I am too using Mozilla Firefox.

Are you maybe using an older browser version? Do you see any errors in your browser's console?

**Comment by dkcodec** [comment]
I guess that the problem is in your internet connection and also maybe in the browser. Imho, Firefox is not the best for GitHub

Try Chrome or Edge

**Reply by WojciechMigda** [reply]
Chrome and Edge are a big no no. And don't get me started on github trying to enforce what browser and what version must be used for the task of CODE HOSTING SERVICE. I am on the verge of migrating all my repos to bitbucket or somewhere else because right now github is unusable with the browser that I consciously decided to use for many many years. Microsoft has yet to realize what are the consequences of messing up with open source software development community.

**Comment by WojciechMigda** [comment]
@HyperX14 Do you still observe the issue? For me, as of today, it seems to have been resolved.
